sdk/storage/azure-storage-blob/azure/storage/blob/_shared/base_client.py

```diff
@@ -76,6 +76,7 @@ def __init__(
         self._location_mode = kwargs.get("_location_mode", LocationMode.PRIMARY)
         self._hosts = kwargs.get("_hosts")
         self.scheme = parsed_url.scheme
+        self.is_azurite = None
```

Comment:
Looking for feedback on this name, and if this is how we want to set it (`None` vs `False`)

Reply:
`False`

Reply:
`_is_XXXX`